languages/python/oso/polar/host.py

```diff
@@ -112,6 +114,13 @@ def cache_class(
         )
         return name
 
+    def register_mros(self):
+        """Register the MRO of each registered class to be used for rule prototype validation."""
```

Comment:
you used "prototype" here but changed "prototype" to "type" in a diff file

Reply:
ah
yeah
darn

Reply:
gonna do a whole new PR to change prototype to type so i'm gonna ignore this for now.

Reply:
sgtm